src/content/advanced/cluster-management/node-pools-capi/index.md

```diff
@@ -0,0 +1,322 @@
+
+- Even without spot instances, AWS has a limited number of instances per type in each Availability Zone. It can happen that your selected instance type is temporarily out of stock just in the moment you are in demand of more worker nodes. Allowing the node pool to use multiple instance types reduces this risk and increases the likelihood that your node pool can grow when in need.
+
+## Node pools and autoscaling {#autoscaling}
+
+With node pools, you set the autoscaling range per node pool (suppported on AWS EC2 (CAPA) clusters only). The Kubernetes cluster autoscaler has to decide which node pool to scale under which circumstances.
+
+If you assign workloads to node pools as described [above](#assigning-workloads) and the autoscaler finds pods in `Pending` state, it will decide based on the node selectors which node pools to scale up.
+
+In case there are workloads not assigned to any node pools, the autoscaler may pick any node pool for scaling. For details on the decision logic, please check the upstream FAQ for [AWS](https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/cloudprovider/aws/README.md).
+
+## On-demand and spot instances {#on-demand-spot}
+
+{{< tabs >}}
+{{< tab id="cluster-capa-ec2" for-impl="capa_ec2" >}}
+
+Node pools can make use of [Amazon EC2 Spot Instances](https://aws.amazon.com/ec2/spot/). You just need to enable it and select the max price to pay on the node pool definition.
+
+```yaml
+metadata:
+  description: "my cluster"
+  name: test-cluster
+  organization: giantswarm
+nodePools:
+  pool0:
+    maxSize: 2
+    minSize: 2
+    spotInstances:
+      enabled: true
+      maxPrice: 1.2
+```
+
+{{< /tab >}}
+{{< tab id="cluster-capz-azure-vms" for-impl="capz_vms" >}}
+
+This is currently not supported.
+
+{{< /tab >}}
+{{< /tabs >}}
+
+## Limitations
+
+- Clusters without worker nodes (= without node pools) cannot be considered fully functional. In order to have all
+required components scheduled, worker nodes are required. For that reason, we deactivate any monitoring and alerts for
+these clusters and don't provide any proactive support.
+
+### Things to keep in mind
+
+{{< tabs >}}
+{{< tab id="cluster-capa-ec2" for-impl="capa_ec2" >}}
+
+- At times, the EC2 instance type required by a node pool can be unavailable in certain availability zones. This can
+result in node pools providing less than the desired number of nodes. The more availability zones a node pool spans,
+the less likely this problem is to occur.
+
+- By default, clusters can have up to 5 node pools. This is limited by the AWS service quota named "IPv4 CIDR blocks
+per VPC" in the [VPC section](https://docs.aws.amazon.com/vpc/latest/userguide/amazon-vpc-limits.html). As the AWS
+account owner, you can request an increase of this limit via the AWS console.
+
+- Node pools can span a maximum of four availability zones. This limit affects both the number of availability zones
```

Comment:
TBH I am not sure this is true as well, I think we are limited by number of AZs in the region and also there is setting how many AZs should be used for subnets  https://github.com/giantswarm/cluster-aws/blob/master/helm/cluster-aws/templates/_aws_cluster.tpl#L56

Reply:
Probably this limitation, if it's even applicable, is too detailed and most customers won't hit it. Maybe we want to remove this hint altogether, or keep only the vintage specifics on a different tab.